src/main/c/netty_quic_quiche.c

```diff
@@ -237,6 +237,26 @@ static jboolean netty_quic_quiche_conn_is_closed(JNIEnv* env, jclass clazz, jlon
     return quiche_conn_is_closed((quiche_conn *) conn) == true ? JNI_TRUE : JNI_FALSE;
 }
 
+static jlongArray netty_quic_quiche_conn_stats(JNIEnv* env, jclass clazz, jlong conn) {
+    quiche_stats stats;
```

Comment:
nit: We usually always init... So:

```suggestion
    quiche_stats stats = NULL;
```

Reply:
I added `memset` to 0s. Would that work? 🤔

Reply:
No just make it NULL as I did in my suggestion